suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx

```diff
@@ -60,14 +54,20 @@ export const TransactionDetailData = ({ transaction }: TransactionDetailDataProp
                     <TransactionDetailRow title="Fee">
                         <Box alignItems="flex-end">
                             <Text color="gray1000">
-                                {CryptoAmountFormatter.format(fee, {
-                                    symbol: transaction.symbol,
-                                })}
+                                <CryptoAmountFormatter
+                                    value={transaction.fee}
+                                    symbol={transaction.symbol}
+                                />
                             </Text>
                             <Text variant="hint" color="gray600">
-                                {`≈ ${FiatAmountFormatter.format(fiatFeeAmount ?? 0, {
-                                    currency: fiatCurrency.label,
-                                })}`}
+                                ≈{' '}
+                                <CryptoToFiatAmountFormatter
```

Comment:
are you sure this is right? I think things like network and customRates should be as separate props

Reply:
okay, I will pass it as contextData

Reply:
That's not what I mean. I think formatters was designed to by used like this:

```tsx
<CryptoToFiatAmountFormatter value={amount} network={transaction.symbol} customRates={rates} />
```

Reply:
yeah but that's only done through context data. For this formatter i used different way of sending value and need to change it. I forgot about that